<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Laba333Class</class>
 <widget class="QWidget" name="Laba333Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Laba333</string>
  </property>
  <widget class="QGroupBox" name="add">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>221</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Добавить элемент</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>166</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Модель</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="modelAdd"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Цвет</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="colorAdd"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Значение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="znAdd"/>
     </item>
     <item>
      <widget class="QPushButton" name="addAdd">
       <property name="text">
        <string>добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="get">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>400</y>
     <width>211</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Получить значение</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>19</y>
      <width>191</width>
      <height>169</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="znGet">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Модель</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="modelGet"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Цвет</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="colorGet"/>
     </item>
     <item>
      <widget class="QPushButton" name="getGet">
       <property name="text">
        <string>получить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delGet">
       <property name="text">
        <string>удалить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>351</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>модель</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>цвет</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>значение</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="menu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Меню</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="menuFiles">
       <property name="text">
        <string>файл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="menuAdd">
       <property name="text">
        <string>Добавить значение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="menuGet">
       <property name="text">
        <string>получить значение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="menuGetAll">
       <property name="text">
        <string>получить все значения</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="files">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>221</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Файл</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>153</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Чтеие/Запись в файл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameFiles">
       <property name="text">
        <string>ffff.txt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addFile">
       <property name="text">
        <string>считать из файла</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="printFile">
       <property name="text">
        <string>выгрузить в файл</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="delAllFiles">
       <property name="text">
        <string>удалить все</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="laba333.qrc"/>
 </resources>
 <connections/>
</ui>
